tests/particle_interpolator_bilinear_least_squares_solkz/screen-output

```diff
@@ -1,19 +1,19 @@
 -----------------------------------------------------------------------------
 -----------------------------------------------------------------------------
 
-Loading shared library <./libparticle_interpolator_bilinear_solkz.so>
+Loading shared library <./libparticle_interpolator_bilinear_least_squares_solkz.so>
 
 Number of active cells: 1,024 (on 6 levels)
 Number of degrees of freedom: 32,196 (8,450+1,089+4,225+9,216+9,216)
 
 *** Timestep 0:  t=0 years
    Skipping temperature solve because RHS is zero.
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 46+0 iterations.
+   Solving Stokes system... 47+0 iterations.
 
    Postprocessing:
-     Writing particle output:       output-particle_interpolator_bilinear_solkz/particles/particles-00000
-     Errors u_L1, p_L1, u_L2, p_L2: 2.396693e-07, 3.813761e-04, 3.798509e-07, 1.430329e-03
+     Writing particle output:       output-particle_interpolator_bilinear_least_squares_solkz/particles/particles-00000
+     Errors u_L1, p_L1, u_L2, p_L2: 1.768545e-08, 1.580101e-04, 3.863311e-08, 2.648933e-04
```

Comment:
Is this the same test change that occured in #1700? Please check that your test actually uses the particle properties for the material model. If so, is this change of results by chance? Or is the difference between the particle and the compositional fields case exactly caused by the over-/undershoot in the interpolation scheme?